source/lmp/pair_deepmd.cpp

```diff
@@ -902,6 +1041,8 @@ static bool is_key(const string &input) {
   keys.push_back("atomic");
   keys.push_back("relative");
   keys.push_back("relative_v");
+  keys.push_back("virtual_len");
+  keys.push_back("spin_norm");
```

Comment:
Please add a test to `source/lmp/tests` for new keys.

Reply:
We have changed source code of LAMMPS. So, We can't provide a unit test directly.